wranglerjs/index.js

```diff
@@ -54,6 +54,16 @@ function filterByExtension(ext) {
   }
   config.target = "webworker";
 
+  if (config.devtool !== undefined && config.devtool !== "inline-source-map") {
```

Comment:
i'm not sure if we want to be this opinionated about source maps - i'd say we could throw a warning saying that source maps are highly recommended but i'm not quite sure we want to tell devs that they are wrong for not using source maps.

Reply:
especially given the size constraints we put on our customers

Reply:
I believe source-maps will always be accounted in the bundle size, because they will ship along with the worker somehow.

Reply:
yah the most obvious way to send source maps along with a worker without tacking on kv (and the latency involved) would be to include them in the bundle as text blob bindings.